bigquery/samples/tests/conftest.py

```diff
@@ -25,6 +25,20 @@ def client():
     return bigquery.Client()
 
 
+@pytest.fixture
+def project_id(client):
```

Comment:
Since you don't use the `project_id` fixture anywhere, this can be removed.

Reply:
Removed!